docs/photoniq/p3/manage-p3-policies.md

```diff
@@ -55,6 +48,15 @@ Perform the following steps to create a new P3 policy:
       or provide regex:
         - ^/us/category/.*$
         - ^/us/product/.*$
+   - **Sample Path** - NEED DEFINITION
```

Comment:
@dwivedianurag - @nobuyoshi-macrometa explained that this is a URL that needs to match the regex pattern, but I do not understand why as a user I am entering this as well as the pattern.